web/src/main/java/DTO/BookDTO.java

```diff
@@ -0,0 +1,104 @@
+package DTO;
+
+import entities.Library;
+import lombok.*;
+
+import java.io.Serializable;
+import java.sql.Timestamp;
+
+@AllArgsConstructor
+@NoArgsConstructor
+public class BookDTO implements Serializable {
```

Comment:
You can easily write @Data once on the class level and the effect will be the same